examples/tiled-rendering-2d_.py

```diff
@@ -24,8 +24,8 @@
 # important: if this is not set, the entire ~4GB array will be created!
 os.environ.setdefault('NAPARI_OCTREE', '1')
```

Comment:
does this have to be before import of napari? Otherwise, we can move it below to avoid the import lint errors.

Reply:
Yes. It need to be before. Currently octotree is resolved on startup.

It is weakpoint of napari bu out of this PR  scope.